src/codeVerify.js

```diff
@@ -0,0 +1,13 @@
+const codeGen = require('./codeGen.js');
+
+function verifyCode() {
+  const userCode = document.getElementById('userCode1').value;
+  const found = false;
+  if ((userCode === (secret))) {
+  // if the code and user code are a match, send a alert
+    util.log('You are correct this is the code');
+  } else if (found === false) {
+    util.log('Wrong code Please try again');
```

Comment:
The util.log method accepts three parameters. Review the documentation and the FoundationsApplied.md lecture file for more info.